src/redux/configureStore.js

```diff
@@ -0,0 +1,12 @@
+import { configureStore, combineReducers } from '@reduxjs/toolkit/dist';
```

Comment:
- [ ] @lucascomamala you are supposed to import configureStore from @reduxjs/toolkit not @reduxjs/toolkit/dist.